spec/v2/payload_sanitizer_spec.rb

```diff
@@ -155,6 +159,71 @@ class CustomConverterWithProvidedValue < InputSanitizer::V2::PayloadSanitizer
     end
   end
 
+  describe "strip_4byte_chars option" do
+    it "is valid when given a string with 4-byte chars" do
+      @params = { :utf8mb4_string => " test \u{1F435} value " }
+      sanitizer.should be_valid
+    end
+
+    it "returns sanitized string without 4-byte chars" do
+      @params = { :utf8mb4_string => " test \u{1F435} value " }
```

Comment:
```suggestion
      @params = { :utf8mb4_string => " test \u{1F435} value !\u{1F435}" }
```

Reply:
minor: the suggestion is an idea to add something that doesn't require the reader to count spaces to understand whetaher the 4-byt char has been removed or changed into space. I looked at the expectation and needed to count spaces to fululy grasp it